sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -98,17 +105,28 @@ impl OpenOcdServer {
             args.join(" ")
         );
 
-        // TODO: forward output/stderr to loggin using a worker, like for verilator.
         let mut cmd = Command::new(&self.opts.openocd);
         cmd.args(args)
             .stdin(Stdio::null())
             .stdout(Stdio::inherit())
             .stderr(Stdio::inherit());
-        let child = cmd.spawn().with_context(|| {
+        let mut child = cmd.spawn().with_context(|| {
             let program = cmd.get_program();
             let args = cmd.get_args().collect::<Vec<_>>().join(OsStr::new(" "));
             format!("failed to spawn openocd: {program:?} {args:?}",)
         })?;
+        // printer stdout and stderr
+        let stdout = child.stdout.take().unwrap();
+        let stderr = child.stderr.take().unwrap();
+        let a = self.openocd_accumulated_stdout.clone();
+        let b = self.openocd_accumulated_stderr.clone();
+        std::thread::spawn(move || {
+            printer::accumulate(stdout, "opentitanlib::util::openocd::stdout", a)
```

Comment:
should we use `concat!(module_path!(), "::stdout")` instead of hardcoding the string in case this code every moves around? (same below)

Reply:
fixed.